packages/datetime/src/_daterangepicker.scss

```diff
@@ -63,20 +63,45 @@ Styleguide components.datetime.daterangepicker.js
     margin-right: $pt-grid-size;
   }
 
+  .DayPicker-Day--hovered-range:not(.DayPicker-Day--outside) {
+    border-radius: 0;
+
+    // need to disable hover-styles for all variants of selected dates
+    // stylelint-disable max-line-length
+    &:not(.DayPicker-Day--selected):not(.DayPicker-Day--selected-range):not(.DayPicker-Day--selected-range-start):not(.DayPicker-Day--selected-range-end) {
+      background-color: $light-gray4;
+
+      &:hover {
+        background-color: $light-gray1;
+      }
+    }
+    // stylelint-enable max-line-length
+  }
+
   .DayPicker-Day--selected-range:not(.DayPicker-Day--outside) {
     border-radius: 0;
     background-color: $light-gray4;
+
+    &:hover {
+      background-color: $light-gray1;
+    }
+  }
+
+  .DayPicker-Day--outside {
```

Comment:
I've hidden "outside" dates (dates in Month N's calendar that are actually part of Month N±1) for a few reasons:

1. When you click on an outside date, it selects the corresponding date in the neighboring month. To show an accurate preview on hover, you'd presumably want to highlight those actual dates that would be included in the selection post-click. But this feels super weird. Look:

![2017-01-31 17 27 52](https://cloud.githubusercontent.com/assets/443450/22491798/c1891bf8-e7da-11e6-8f03-b47cf7baa81b.gif)

Granted, there _is_ a missing style revealed by this GIF: a hovered-range endpoint that isn't selected or below the mouse cursor. Even with that, I think this would feel weird.

2. It's not clear whether you should highlight an outside date when it falls within a hovered range. I got feedback from other designers in 101F that they were confused that the outside dates weren't also highlighted when a hovered range spanned them:

![image](https://cloud.githubusercontent.com/assets/443450/22491891/70ba3512-e7db-11e6-98c8-a83e1e4395de.png)

3. (Not really relevant to this PR's scope) We may eventually consider adding a "disabled" style to dates that precede a selected start date to prevent the selection of invalid, earlier end dates. I'd want those disabled dates to look exactly the way outside dates look now.